src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
I don’t know why we’re getting into such pedantic name arguments here,
especially given that I’m the one expert on these concepts and the sole
author of the corresponding papers/notes.  Moreover, the interfaces can 
do whatever they want with the names — as established consistency is 
no longer a concern.  

I want to get working on a bunch of other changes that are waiting on this 
so I made the change.  The Windows tests are failing due to what looks
like a bad configuration — given how terribly this has gone I hope this
failure will be addressed promptly.

Having a PR wait this long for such pedantic concerns is absolutely not 
sustainable.  

On Mar 14, 2016, at 4:34 AM, Bob Carpenter <notifications@github.com> wrote:

> In src/stan/mcmc/hmc/nuts/base_nuts.hpp:
> 
> > @@ -152,14 +152,16 @@ namespace stan {
> >          names.push_back("stepsize__");
> >          names.push_back("treedepth__");
> >          names.push_back("n_leapfrog__");
> > -        names.push_back("n_divergent__");
> > +        names.push_back("divergent__");
> > +        names.push_back("E__");
> It's easy for a user who has read Michael's papers to map energy__ to E__, but the reverse step is a little more confusing. On the other hand, just seeing energy__ isn't going to help most of our users without going to Michael's papers. So I'd prefer energy__ to match the other params and be a little less confusing, but I don't think it's a huge deal either way. Narrower column names do have advantages. - Bob
> …
> —
> Reply to this email directly or view it on GitHub.
>